tools/cbmc/proofs/ARP/ARPAgeCache/cbmc-batch.yaml

```diff
@@ -1,4 +0,0 @@
-jobos: ubuntu16
```

Comment:
I know that we have to remove the cbmc-batch.yaml files from the repository, but I think this should be done in another PR. I don't see how they interfere with the pure back porting.
There are no paths in the cbmc-batch.yaml files.

Reply:
Thank you, I'm keeping all `cbmc-batch.yaml` files for this PR now.